### PR TITLE
[To Vector] add bitmap in tablet

### PR DESCRIPTION
**Pull request by CRZbulabula**
using java class BitSet

### Content3 ...

<hr>

This PR has:
- [ ] been self-reviewed.
    - [ ] concurrent read
    - [ ] concurrent write
    - [ ] concurrent read and write 
- [ ] added documentation for new or modified features or behaviors.
- [ ] added Javadocs for most classes and all non-trivial methods. 
- [ ] added or updated version, __license__, or notice information
- [ ] added comments explaining the "why" and the intent of the code wherever would not be obvious 
  for an unfamiliar reader.
- [ ] added unit tests or modified existing tests to cover new code paths, ensuring the threshold 
  for code coverage.
- [ ] added integration tests.
- [ ] been tested in a test IoTDB cluster.

<hr>

### CODE REVIEW

File: session/src/main/java/org/apache/iotdb/session/SessionUtils.java [hunk 1]
```diff
@@ -58,42 +59,95 @@ public static ByteBuffer getValueBuffer(Tablet tablet) {
...
+        while (true) {
+          nextPos = curBitset.nextClearBit(lastPos);
+          for (int index = lastPos; index < nextPos; index++) {
+            valueBuffer.putInt(intValues[index]);
+          }
+          if (nextPos == tablet.rowSize) {
+            break;
+          }
+          valueBuffer.putInt(-1);
+          lastPos = nextPos + 1;
```
Comment (HTHou) [inline session/src/main/java/org/apache/iotdb/session/SessionUtils.java:77]: Is there any way to put the bitSet into valueBuffer? The server needs to get the bitSet to know which value is valid....

**Comment by jixuan1989** [comment]
you can try another bitmap implementation https://github.com/apache/iotdb/blob/encoding_parallel/server/src/main/java/org/apache/iotdb/db/utils/BitMap.java

compare its performance with BitSet.

**Comment by CRZbulabula** [comment]
> you can try another bitmap implementation https://github.com/apache/iotdb/blob/encoding_parallel/server/src/main/java/org/apache/iotdb/db/utils/BitMap.java
> 
> compare its performance with BitSet.

I've tried to implement bitmap myself, but the bitset in Java.util is more efficient. I'll compare this one tomorrow.

**Comment by CRZbulabula** [comment]
The comparison between bitset and bitmap is shown below

| |  BitSet(Set) | BitSet(Get) | BitSet(Clear) | BitMap(Set) | BitMap(Get) | BitMap(Clear) |
|  ---- | ---- | ---- | ---- | ---- | ---- | ---- |
| 3200(500000) | 2006 | 2045 | 8 | 1489 | 555 | 5 |
| 32000(50000) | 2964 | 2068 | 7 | 1571 | 548 | 2 |
| 320000(5000) | 2459 | 2040 | 6 | 1643 | 602 | 3 |
| 3200000(500) | 3042 | 1637 | 17 | 1454 | 630 | 7 |
| 32000000(50) | 3081 | 1664 | 18 | 1491 | 616 | 20 |

**Pull request merged**